documentation/asciidoc/accessories/iqaudio/introduction.adoc

```diff
@@ -1,30 +1,29 @@
-Balanced Out:: An XLR connector - Normally a 3 pin male connector. This is used in a studio set-up and some
-high-end Hi-Fi systems. It can also be used to drive active “monitor” speakers as used at discos or on
+Line out:: A double phono/RCA connector, normally red and white in colour. This output is a variable
+analogue signal (0.2V RMS) and can connect to your existing hi-fi (pre-amp or amplifier), or can be used
+to drive active speakers which have their own amplifier built in.
+Balanced out:: An XLR connector, normally a three-pin male connector. This is used in a studio set-up, and in some
+high-end hi-fi systems. It can also be used to drive active monitor speakers like those used at clubs or on
 stage directed towards the DJ or performers. 
-Stereo Speakers:: 2 sets of screw terminals for 2 x 25W speakers. These are for normal Hi-Fi speakers such as bookshelf or floor
-standing. These are known as passive speakers.
-Mono Speaker:: Screw terminal for single 1.2W speaker, as found in a transistor radio or similar.
-Headphones:: 3.5mm jack socket delivering stereo audio for a set of normal headphones. Our headphone amplifiers on the DAC boards can drive up to 80/90ohm impedance headphones. The headphone output of the Codec+ is also aux out.
-Aux In:: A double Phono/RCA connector or 3.5mm socket. Accepts analogue audio in up to 1V RMS. This can be used to record audio from a variable analogue source such as a mobile phone, MP3 player or similar.
-Aux Out:: A double Phono/RCA connector or 3.5mm socket. Delivers analogue audio out up to 1V RMS. This can be used to feed audio into an amplifier at a reduced volume compared to Line Out.
-Mic In:: A 3.5mm socket for use with an external Electret microphone. The built-in MEMS microphone on the Codec Zero is automatically disabled when the External Mic In connector is used.
+Stereo speakers:: Two sets of screw terminals for 2×25W speakers. These are for traditional hi-fi speakers without built-in amplification. These are known as passive speakers.
+Mono speaker:: A screw terminal for a single 1.2W speaker, as found in a transistor radio or similar.
+Headphones:: A 3.5mm jack socket delivering stereo audio for a set of headphones. The headphone amplifiers on the Raspberry Pi DAC boards can drive up to 80/90Ω impedance headphones. The headphone output of the Codec+ is also aux out.
+Aux in:: A double Phono/RCA connector or 3.5mm socket. Accepts analogue audio in up to 1V RMS. This can be used to record audio from a variable analogue source such as a mobile phone, MP3 player or similar.
+Aux out:: A double Phono/RCA connector or 3.5mm socket. Delivers analogue audio out up to 1V RMS. This can be used to feed audio into an amplifier at a reduced volume compared to Line out.
+Mic in:: A 3.5mm socket for use with an external electret microphone. The built-in MEMS microphone on the Codec Zero is automatically disabled when the external Mic in connector is used.
```

Comment:
All the definitions in this list match the table column-headers, except for this last one where the table says "Ext mic" and the list here says "Mic in" ?

Reply:
Point.

Reply:
Fixed